app/models/profile.rb

```diff
@@ -187,6 +187,18 @@ def activate_after_password_reset
     end
```

Comment:
It looks like `activate_after_password_reset` is now unused and should be removed.

Reply:
I'm going to resolve this comment because Shane deleted `activate_after_password_reset` in a follow-up PR.